### PR TITLE
Feat Live 10310 live config lib timer fetch

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

Feature flag improvement : 
1. Before this PR, Config is retrieved from firebase only once when LLD/LLM is started. After this PR, config will be retrieved from firebase every 12 hours. 
2. Before this PR, if firebase server is down and there is no local cache, all the config are disabled. After this PR, if firebase server is down and there is no local cache, default values will be taken into account.

### ❓ Context

- **JIRA or GitHub link**: 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [X] `npx changeset` was attached.
- [X] **Covered by automatic tests.** Very hard to write test as we need the mock the firebase service.
- [X] **Impact of the changes:** Config will be retrieved from firebase every 12 hours. If firebase server is not available, use the config from local cache. If the local cache is not available, use the default values.
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx [hunk 1]
```diff
@@ -9,32 +9,33 @@ export const FirebaseRemoteConfigProvider = ({ children }: Props): JSX.Element |
...
-        setLoaded(true);
+      } finally {
+        if (!unmounted) {
+          setLoaded(true);
+        }
       }
     };
     fetchConfig();
-
+    // 12 hours fetch interval. TODO: make this configurable
```
Comment (lvndry) [inline apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx:35]: Could you create a Jira ticket for this ?

File: apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx [hunk 2]
```diff
@@ -25,33 +25,32 @@ export const FirebaseRemoteConfigProvider = ({ children }: Props): JSX.Element |
       console.error(`Failed to initialize Firebase SDK with error: ${error}`);
       setLoaded(true);
     }
+    const remoteConfig = getRemoteConfig();
+    remoteConfig.settings.minimumFetchIntervalMillis = 0;
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx:29]: Is it safe to set this to 0 even in prod?

Comment (hzheng-ledger) [reply apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx:29]: yes, it is very safe.
even if the firebase server is not accessible or is blocked due to the rate limit, the "getValue" method can get the config from local cache(if the local cache fails, it gets config from default value)

**Review by lvndry — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Comment by hzheng-ledger** [comment]
PR has been merged with https://github.com/LedgerHQ/ledger-live/pull/5731

**Pull request closed without merging**